### ISSUE TITLE
loss_tag is too large

### ISSUE THREAD

**Issue by SANJINGSHOU14** [xinyu1205/recognize-anything#136]
Hello, I am pre-training the RAM model with the COCO dataset you offered in this repo.
This is my "pretrain.yaml":
```
train_file: ['/xxx/coco_train_rmcocodev_ram.json',]
image_path_root: "/xx/DATA"
vit: 'swin_l'
vit_grad_ckpt: False
vit_ckpt_layer: 0
image_size: 224
batch_size: 8
weight_decay: 0.005 # 0.005 or 0
init_lr: 1e-4 # larger
min_lr: 5e-6 # larger
warmup_lr: 5e-7
lr_decay_rate: 0.8 # smaller
max_epoch: 10
warmup_steps: 3000
class_num: 4585
```
But the `loss_tag` keeps around `90` when training is finished.
Could you please give me some suggestions to fix the problem?
Thanks in advance!

**Comment by xinyu1205** [comment]
This is normal, since the loss is taken as [sum()](https://github.com/xinyu1205/recognize-anything/blob/main/ram/models/utils.py#L326), instead of mean().
